ngraph/core/src/function.cpp

```diff
@@ -347,6 +370,38 @@ bool Function::visit_attributes(AttributeVisitor& visitor)
     return true;
 }
 
+void Function::add_sinks(const SinkVector& sinks)
+{
+    m_sinks.insert(m_sinks.begin(), sinks.begin(), sinks.end());
```

Comment:
I guess we should insert at the end of the m_sinks vector
m_sinks.insert(m_sinks.begin(), sinks.begin(), sinks.end()); -> m_sinks.insert(m_sinks.end(), sinks.begin(), sinks.end());

Reply:
fixed